include/tvm/tir/schedule/schedule.h

```diff
@@ -277,19 +277,22 @@ class ScheduleNode : public runtime::Object {
    * 3) All loops must start with 0.
    * 4) The domain of a loop to be fused cannot depend on another loop to be fused.
    * \param loop_rvs The loops to be fused
+   * \param preserve_unit_iters Whether or not to preserve unit iterators in block bindings
    * \return The new loop after fusion
    */
-  virtual LoopRV Fuse(const Array<LoopRV>& loop_rvs) = 0;
+  virtual LoopRV Fuse(const Array<LoopRV>& loop_rvs, bool preserve_unit_iters = true) = 0;
```

Comment:
I want to change the default value to false for better useability.

Reply:
Usually it doesn't affect usability if we don't over-simplify anything though...but i don't have strong opinion on what a good default should be

Reply:
I mean, I don't want to change the default behavior and user interface here. I.e., users already have a schedule, but it may fail after syncing with upstream since there are additional loops and influence `get_loops`.

Reply:
Hey it's not related to number of loops actually. Updated the PR description for better clarity.

Reply:
Will this change of default behavior affect unit tests? I'm not sure whether the tests of split/fuse will fail because the bindings are not structural equal .

Reply:
In my opinion, it's always good to keep the original default behavior.

Reply:
okay I’m not opinionated - will set this to False as the default behavior.

Reply:
OK, it's my misunderstanding. Default true looks look to me now. Thanks @junrushao1994 for the explaination.